corehq/apps/reports/views.py

```diff
@@ -1304,6 +1305,31 @@ def page_context(self):
         }
 
 
+def form_to_json(domain, form, timezone=None):
+    form_name = xmlns_to_name(
+        domain,
+        form.xmlns,
+        app_id=form.app_id,
+        lang=get_language(),
+    )
+    if timezone is None:
+        received_on = json_format_datetime(form.received_on)
+    else:
+        offset = timezone.utcoffset(datetime.utcnow())
+        local_received_on = form.received_on + offset
+        received_on = local_received_on.strftime("%Y-%m-%d %H:%M")
```

Comment:
why not use `json_format_datetime` to be consistent with above?

Reply:
The "forms" page that uses the `json_format` formats the datetime for display using javascript which ends up outputting the same thing. I wasn't passing any of my values through js, and it seemed easier to just do this in python. 

https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/casexml/apps/case/static/case/js/case_details.js#L14-L42